odoo/addons/base/models/res_bank.py

```diff
@@ -12,68 +10,12 @@ def sanitize_account_number(acc_number):
     return False
 
 
-class ResBank(models.Model):
```

Comment:
Keep it normalized. Why do you want to repeat all that information for each user?